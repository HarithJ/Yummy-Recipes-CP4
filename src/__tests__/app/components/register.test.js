import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Register from '../../../components/auth/register/register.js'

describe("register component", () => {
  const register = shallow(<Register />)

  it("renders properly", () => {
    expect(shallowToJson(register)).toMatchSnapshot();
  })
})
