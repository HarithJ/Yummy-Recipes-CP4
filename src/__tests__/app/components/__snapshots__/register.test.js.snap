// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`register component renders properly 1`] = `
<div>
  <Nav
    link="/login"
    linkTitle="Login"
  />
  <Header />
  <RegistrationForm />
</div>
`;
